aptos-move/aptos-aggregator/src/transaction.rs

```diff
@@ -31,19 +33,19 @@ impl AggregatorValue {
 pub struct ChangeSetExt {
     delta_change_set: DeltaChangeSet,
     change_set: ChangeSet,
-    gas_feature_version: u64,
+    checker: Arc<dyn CheckChangeSet>,
```

Comment:
I still believe that the config/checker should be owned by the process (Aptos VM or more broadly the executor) instead of the data it's checking, but considering that this particularly change isn't really making the problem worse (the gas_feature_version used to perform similar roles), I'm fine with us addressing this in a later refactoring, which we'll have to do sooner or later.